src/pages/index.js

```diff
@@ -7,6 +6,8 @@ import * as constants from "../utils/constants.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import "../pages/index.css";
+import Api from "../components/Api.js";
+import PopupDeleteCard from "../components/PopupDeleteCard.js";
 
 /* Functions */
 function handleEsc(evt) {
```

Comment:
It's not needed anymore

Reply:
Thanks for the feedback! Can I get clarification on this particular part? I'm not seeing what's "not needed anymore".

Reply:
Hello! I'm not seeing a required correction here so I'm going to push and resubmit my project for review. Thanks!

Reply:
`handleEsc` is not needed anymore in `index.js`